src/cuckoocache.h

```diff
@@ -237,8 +234,8 @@ class cache
      *  32*32->64 multiply, which means the operation is reasonably fast even on a
      *  typical 32-bit processor.
      *
-     * @param e the element whose hashes will be returned
-     * @returns std::array<uint32_t, 8> of deterministic hashes derived from e
+     * @param e The element whose hashes will be returned
+     * @returns Deterministic hashes derived from `e`
```

Comment:
maybe good to update to say "Deterministic hashes derived from `e` uniformly mapped onto the range `[0, size)`.